circle.yml

```diff
@@ -37,9 +37,6 @@ test:
   override:
     - ./bin/circleci/test-addon.sh
     - ./bin/circleci/test-frontend.sh
-    - sh ./bin/circleci/test-firefox-release.sh
```

Comment:
We have a history of turning integration tests off and forgetting about them. Should at least leave the tests for Firefox release enabled and plan to improve things to the point where we can turn the other two back on.